CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Added
+* Support addition of "Extract" to layer info capabilities if provider's geojson includes `{ capabilities: { extract: true } }`
```

Comment:
need to doc the capability in a few places including readme here

Reply:
Added in https://github.com/koopjs/FeatureServer/pull/120/commits/d43a56315f5e2312961f0939290c4ea56dc26e78.  Will add to koopjs.github.io as well.

Reply:
Actually - maybe a description of the `capabilities` properties should be left out of the provider spec since these are ArcGIS client specific?